src/extractor/extractor.cpp

```diff
@@ -359,18 +362,6 @@ void Extractor::FindComponents(unsigned max_edge_id,
         }
     }
 
-    // connect forward and backward nodes of each edge
```

Comment:
@TheMarex just forgot to mention this place that was introduced in c80c2233c5a537696e194c55b6f4f9cf162b4875. i don't get idea why graph must be extended with potentially non-exisiting edges. So if there is a use case for it must be  reverted before merge